CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ If you have questions about how to use a component or element in Collab-UI, plea
 ### <a name="issue"></a> Found an Issue?
 
 If you find a bug in the source code or a mistake in the documentation, you can help us by
-submitting an issue to our [GitHub Repository](https://github.com/collab-ui/collab-ui/issues).  Even better you can submit a Pull Request
+submitting an issue to our [GitHub Repository](https://github.com/momentum-design/momentum-ui/issues).  Even better you can submit a Pull Request
```

Comment:
Will we be moving to new Github org too?

Reply:
yes. As soon as this merges, I will be renaming the org and repo.